### PR TITLE
Ports a few tg foods

### PR DESCRIPTION
**Pull request by tristan1333**
## What Does This PR Do
Ports a few tg kitchen recipes:  BBQ ribs, BBQ sauce, appendix burger, bacon burger, bearger, five alarm burger, mcguffin, mcrib

Recipes are the same except for

- BBQ Ribs:  2 meat, 1 rod, 5 BBQ Sauce
- BBQ Sauce: (10 ash, 10 tomato juice, 1 blackpepper) -> 10 BBQ Sauce
-Bacon Burger: Bun, 2 bacon, 1 cheese wedge

## Why It's Good For The Game
More stuff for the chef to cook
## Images of changes
image - Bacon burger
 image - Appendix burger
image  - Bearger
image - Five Alarm burger
image - mcguffin
image - mcRib
image - BBQ ribs
image - BBQ sauce

## Testing
Cooked all items ingame and tested that bbq sauce is makeable

## Changelog
:cl:
add: Ports new recipes from TG station
/:cl:

### CODE REVIEW

File: code/modules/reagents/chemistry/recipes/food_reactions.dm [hunk 1]
```diff
@@ -43,6 +43,13 @@
 	required_reagents = list("soymilk" = 1,"sodiumchloride" = 1, "water" = 8)
 	result_amount = 10
 
+/datum/chemical_reaction/bbqsauce
+	name = "BBQ Sauce"
+	id="bbqsauce"
```
Comment (DGamerL) [inline code/modules/reagents/chemistry/recipes/food_reactions.dm:48]:
```suggestion
	id = "bbqsauce"
```

File: code/modules/food_and_drinks/food/foods/sandwiches.dm [hunk 2]
```diff
@@ -206,3 +206,57 @@
...
+	desc = "The perfect combination of all things American."
+	icon_state = "baconburger"
+	tastes = list("bun" = 1, "bacon" = 1)
+	list_reagents = list("nutriment" = 3, "vitamin" = 2, "protein" = 6)
+	filling_color = "#F2B6EA"
+	bitesize = 3
+
+/obj/item/reagent_containers/food/snacks/bearger
+	name = "bearger"
+	desc = "Best served rawr."
```
Comment (DGamerL) [inline code/modules/food_and_drinks/food/foods/sandwiches.dm:230]: This description is awful. I love it.

File: code/modules/food_and_drinks/food/foods/meat.dm [hunk 3]
```diff
@@ -533,3 +533,11 @@
 	desc = "An appendix which appears to be inflamed."
 	icon_state = "appendixinflamed"
 	filling_color = "#E00D7A"
+
+/obj/item/reagent_containers/food/snacks/bbqribs
+	name = "bbq ribs"
+	desc = "Sweet, Smokey, Savory, and gets everywhere. Perfect for Grilling."
```
Comment (DGamerL) [inline code/modules/food_and_drinks/food/foods/meat.dm:539]:
```suggestion
	desc = "Sweet, smokey, savory, and gets everywhere. Perfect for Grilling."
```

File: code/modules/food_and_drinks/food/condiment.dm [hunk 4]
```diff
@@ -168,6 +169,13 @@
 	list_reagents = list("flour" = 30)
 	possible_states = list()
 
+/obj/item/reagent_containers/food/condiment/bbqsauce
+	name = "bbq sauce"
```
Comment (DGamerL) [inline code/modules/food_and_drinks/food/condiment.dm:173]: Same on this as the other BBQ stuff

**Review by DGamerL — COMMENTED** [review]
Some small things

File: code/modules/food_and_drinks/recipes/recipes_microwave.dm [hunk 5]
```diff
@@ -140,6 +140,64 @@
...
+/datum/recipe/microwave/fivealarmburger
+	items = list(
+		/obj/item/reagent_containers/food/snacks/bun,
+		/obj/item/reagent_containers/food/snacks/grown/ghost_chili,
+		/obj/item/reagent_containers/food/snacks/grown/ghost_chili
+		)
+	result = /obj/item/reagent_containers/food/snacks/fivealarmburger
+
+/datum/recipe/microwave/baconburger
+	items = list(
```
Comment (Henri215) [inline code/modules/food_and_drinks/recipes/recipes_microwave.dm:173]: Any reason why it requires so much bacon and cheesewedge? I have been eating bacon burgers the wrong way my entire life?

Comment (tristan1333) [reply code/modules/food_and_drinks/recipes/recipes_microwave.dm:173]: Just straight ported from tg, can reduce to whatever is reasonable

Comment (Henri215) [reply code/modules/food_and_drinks/recipes/recipes_microwave.dm:173]: Ask what others think about this amount, i am not that knowledgeable with recipes to know if its too much.

Comment (DGamerL) [reply code/modules/food_and_drinks/recipes/recipes_microwave.dm:173]: For a burger like this, I'd probably knock it down to two pieces of bacon and one cheese wedge

Comment (tristan1333) [reply code/modules/food_and_drinks/recipes/recipes_microwave.dm:173]: doneskis

File: code/modules/food_and_drinks/recipes/recipes_microwave.dm [hunk 6]
```diff
@@ -140,6 +140,61 @@
...
+		/obj/item/reagent_containers/food/snacks/bun,
+		/obj/item/organ/internal/appendix
+	)
+	result = /obj/item/reagent_containers/food/snacks/appendixburger
+
+/datum/recipe/microwave/appendixburger_bitten
+	items = list(
+		/obj/item/reagent_containers/food/snacks/appendix,
+		/obj/item/reagent_containers/food/snacks/bun
+		)
```
Comment (Henri215) [inline code/modules/food_and_drinks/recipes/recipes_microwave.dm:154]: Adujst the identation for all the recipes items please (look other recipes in this file and follow their style).

**Review by Henri215 — COMMENTED** [review]
Update your PR description since the bacon burger recipe is no longer the same as the one in TG.

File: code/modules/food_and_drinks/recipes/recipes_microwave.dm (hunk 6 above)
Comment (tristan1333) [reply code/modules/food_and_drinks/recipes/recipes_microwave.dm]: should be cleared up now

**Review by Henri215 — APPROVED** [review]
LGTM (Lets Go To Mexico)

**Review by S34NW — APPROVED** [review]
deconflict for merge

**Review by hal9000PR — APPROVED** [review]

**Pull request merged**